README.md

```diff
@@ -339,9 +340,17 @@ How to choose between all these features? Here are some considerations:
         #[serde(with = "clickhouse::serde::time::date")]
         date: Date,
     }
+
+    // or if you want to use `chrono`: 
+      #[derive(Row, Serialize, Deserialize)]
+    struct MyRow {
+        days: u16,
+        #[serde(with = "clickhouse::serde::chrono::date")]
+        date: NaiveDate,
+    }
     ```
     </details>
-* `Date32` maps to/from `i32` or a newtype around it and represents a number of days elapsed since `1970-01-01`. Also, [`time::Date`](https://docs.rs/time/latest/time/struct.Date.html) is supported by using `serde::time::date32`, that requires the `time` feature.
+* `Date32` maps to/from `i32` or a newtype around it and represents a number of days elapsed since `1970-01-01`. Also, [`time::Date`](https://docs.rs/time/latest/time/struct.Date.html) is supported by using `serde::time::date32`, that requires the `time` feature. `chrono::NaiveDate` is supported by using `serde::chrono::date`, requiring the `chrono` feature. 
```

Comment:
Let's rephrase it using a list with all variants.

Also, to be consistent, we need to provide a link to these types (as for time-based types).